doc/source/ray-core/scheduling/accelerators.rst

```diff
@@ -38,6 +38,11 @@ Starting Ray Nodes with Accelerators
 ------------------------------------
 
 By default, Ray will set the quantity of accelerator resources of a node to the physical quantities of accelerators auto detected by Ray.
+
+For GPU auto-detection, Ray uses the `gpustat <https://github.com/wookayin/gpustat>`_ tool.
+If you find that GPUs are not being auto-detected, first test that there are no errors by running ``gpustat --debug``.
+Run this command inside the Ray container if using Docker.
```

Comment:
We no longer use `gpustat` for Nvidia GPU auto-detection but directly using pynvml for Nvidia GPU (see nvidia_gpu.py) and other tools for AMD GPU and Intel GPUs.

Reply:
Hmm I see, can you advise on what users should run to debug the autodetection?

Reply:
@stephanie-wang the current autodetection code is in `nvidia_gpu.py` specifically

```
@staticmethod
    def get_current_node_num_accelerators() -> int:
        import ray._private.thirdparty.pynvml as pynvml

        try:
            pynvml.nvmlInit()
        except pynvml.NVMLError:
            return 0  # pynvml init failed
        device_count = pynvml.nvmlDeviceGetCount()
        pynvml.nvmlShutdown()
        return device_count
```

This is also what `gpustat` uses under the hood.